glue_ginga/plugins/Glue.py

```diff
@@ -134,18 +219,23 @@ def put_data_cb(self):
         kwargs = {name: data_np}
 
         try:
+            # TODO: Pass in WCS for image.
+            #if isinstance(image, AstroImage.AstroImage):
+            #    ???.coords.wcs = image.wcs???
```

Comment:
But you can just extract the astropy wcs from the ginga wrapper.  It is accessible as `image.wcs.wcs`